docs/frameworks/laravel.md

```diff
@@ -73,6 +73,15 @@ SESSION_DRIVER=array
 LOG_CHANNEL=stderr
 ```
 
+Next we need to use the new `VIEW_COMPILED_PATH` env var when it has been set. Update the `config/view.php` file:
+
+```php
+    'compiled' => env(
+        'VIEW_COMPILED_PATH',
+        realpath(storage_path('framework/views'))
+    ),
+```
```

Comment:
Thank you. This piece of code however already exists in Laravel: https://github.com/laravel/laravel/blob/master/config/view.php#L31-L34

Are you sure you are not using an older version?